aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -177,7 +177,7 @@ class QConvPackWeightInt8 final : public c10::OperatorKernel {
     if (bias_in.has_value()) {
       bias_fp32 = bias_in.value();
     } else {
-      bias_fp32 = at::zeros(out_ch, at::kFloat);
+      bias_fp32 = at::zeros(out_ch, weight.options().dtype(at::kFloat));
```

Comment:
I patched this PR but it doesn't seem to fix the error on cpp_custom_type_hack() code path...
It still fails at !is_variable() check in empty_cpu:
```
#0  0x00000000006e14de in at::native::empty_cpu(c10::ArrayRef<long>, c10::TensorOptions const&, c10::optional<c10::MemoryFormat>) ()
#1  0x00000000007cf8cf in at::CPUType::empty(c10::ArrayRef<long>, c10::TensorOptions const&, c10::optional<c10::MemoryFormat>) ()
#2  0x00000000007a364d in at::Tensor at::cpp_custom_type_hack::create<PackedConvWeightsQnnp>(std::unique_ptr<PackedConvWeightsQnnp, std::default_delete<PackedConvWeightsQnnp> >, c10::TensorOptions) ()
#3  0x00000000007a2347 in at::native::(anonymous namespace)::QConvPackWeightInt8::qnnpack_conv_prepack(at::Tensor, c10::optional<at::Tensor>, c10::List<long>, c10::List<long>, c10::List<long>, long) [clone .isra.280] ()
```

Reply:
Thanks for the quick feedback. I think this patch is a strict improvement, but I need to kill more spots.